workflows/prognostic_c48_run/tests/test_prepare_config.py

```diff
@@ -83,6 +86,40 @@ def test_config_high_level_duration(duration, expected):
     assert out["namelist"]["coupler_nml"]["seconds"] == expected
 
 
+def test_initial_condition_default_vertical_coordinate_file():
+
+    base_url = "/some/path"
+    timestep = "20160805.000000"
+    test_initial_condition = InitialCondition(base_url, timestep)
+
+    out = test_initial_condition.vertical_coordinate_file
+
+    assert (
+        out == "gs://vcm-fv3config/data/initial_conditions/"
+        "fv_core_79_levels/v1.0/fv_core.res.nc"
+    )
```

Comment:
nit: this might be a little more concise, and make use of the constant defined in `fv3kube`:
```suggestion
    result = InitialCondition(base_url, timestep)
    assert result.vertical_coordinate_file == DEFAULT_VERTICAL_COORDINATE_FILE
```